Client/Frontend/Popover/PopoverController.swift

```diff
@@ -150,6 +154,32 @@ class PopoverController: UIViewController {
         return .lightContent
     }
     
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        
+        switch contentSizeBehavior {
+        case .preferredContentSize, .fixedSize(_):
+            containerView.layoutIfNeeded()
+            if contentController.extendEdgeIntoArrow {
+                contentController.view.frame = containerView.contentView.bounds
+                break
+            }
+            var rect = containerView.contentView.bounds
+            switch containerView.arrowDirection {
+            case .up:
+                rect.origin.y = PopoverUX.arrowSize.height
+                rect.size.height -= PopoverUX.arrowSize.height
```

Comment:
This happens in both .up and .down? If so, it should be moved outside of the switch.